package.json

```diff
@@ -3,35 +3,41 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "node server.js",
```

Comment:
Hello @BrianSerem just for clarity, why did we change this from `ng serve` to `node server.js`

Reply:
This is the start command needed by Heroku. Don't worry, our good old `ng serve` will still work.